.github/workflows/versioned-coverage.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       env:
         VERSIONED_MODE: --major 
         JOBS: 4 # 2 per CPU seems to be the sweet spot in GHA (July 2022)
+        C8: skip
```

Comment:
Should we change this to something like `SKIP_COVERAGE` or `SKIP_C8` to avoid confusion? I've never had a local var and env var named the same in bash, so not sure if that can cause issues, but I do know it's confusing to read in the shell script.

Reply:
sure